library-core/src/main/java/com/mikepenz/iconics/IconicsContentProvider.kt

```
@@ -36,7 +36,7 @@ internal class IconicsContentProvider : ContentProvider() {
     ): Cursor? = null
 
     override fun onCreate(): Boolean {
-        Iconics.init(context!!.applicationContext)
+        Iconics.init(context?.applicationContext)
```

This may hide a silent issue. 

The context should never be null, but if it is, it will result in unexpected behaviour so we should fail, or at least have a log message.

---

If the context is null then when using the !! operator  app will crash. And when using ? operator, it will not crash.

---

? operator was just added to Kotlin to avoid the problem you described

---

Which is anticipated in this case. the `context` should not be null, and if it is, something is terribly wrong. As silently passing null to the `init` will make things fail silently